webapp/src/components/Atlas/Atlas.tsx

```diff
@@ -127,7 +131,7 @@ const Atlas: React.FC<Props> = (props: Props) => {
   )
 
   const forSaleOrRentLayer: Layer = useCallback(
-    (x, y) => {
+    (x: any, y: any) => {
```

Comment:
These should be numbers, right?
```suggestion
    (x: number, y: number) => {
```